in_toto/util_test.go

```diff
@@ -96,15 +96,15 @@ func TestInterfaceKeyStrings(t *testing.T) {
 func TestSubsetCheck(t *testing.T) {
 	tables := []struct {
 		subset   []string
-		superset []string
+		superset Set
 		result   bool
 	}{
-		{[]string{"sha256"}, []string{"sha256", "sha512"}, true},
-		{[]string{"sha512"}, []string{"sha256"}, false},
-		{[]string{"sha256", "sha512"}, []string{"sha128", "sha256", "sha512"}, true},
+		{[]string{"sha256"}, Set{"sha256": {}, "sha512": {}}, true},
+		{[]string{"sha512"}, Set{"sha256": {}}, false},
+		{[]string{"sha256", "sha512"}, Set{"sha128": {}, "sha256": {}, "sha512": {}}, true},
```

Comment:
Minor nit: I think it's a tad more readable if you use e.g. `NewSet{"sha256",  "sha512"}`, etc.